scripts/rollup/bundles.js

```diff
@@ -259,9 +259,7 @@ const bundles = [
 
   /******* React DOM Server *******/
   {
-    bundleTypes: __EXPERIMENTAL__
-      ? [UMD_DEV, UMD_PROD, NODE_DEV, NODE_PROD]
-      : [UMD_DEV, UMD_PROD, NODE_DEV, NODE_PROD, FB_WWW_DEV, FB_WWW_PROD],
+    bundleTypes: [UMD_DEV, UMD_PROD, NODE_DEV, NODE_PROD, FB_WWW_DEV, FB_WWW_PROD],
```

Comment:
The classic version has an override that pulls in from partial renderer.
https://github.com/facebook/react/blob/main/packages/react-dom/src/server/ReactDOMLegacyServerBrowser.classic.fb.js

The modern version has the same config but uses the Fizz legacy implementation.